src/frontend/js/types/Joanie.ts

```diff
@@ -190,7 +190,7 @@ export interface Order {
   id: string;
```

Comment:
Why you do not take opportunity to add the new field `enrollment` ?

Reply:
It's not used yet and i didn't want to add it in case the backend change before we use it in richie.

Reply:
I agree, and what's more, it simplifies the understanding of renaming